app/Http/Livewire/AllProjects.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use App\Enums\EngagementRecruitment;
+use App\Enums\MeetingType;
+use App\Enums\ProvinceOrTerritory;
+use App\Models\DisabilityType;
+use App\Models\Impact;
+use App\Models\Project;
+use App\Models\Sector;
+use Illuminate\Database\Eloquent\Builder;
+use Livewire\Component;
+use Livewire\WithPagination;
+use Spatie\LaravelOptions\Options;
+
+class AllProjects extends Component
+{
+    use WithPagination;
+
+    public string $query = '';
```

Comment:
I think it would be a good idea to change this to `$searchQuery` to differentiate it from all the `$query` calls in the `render()` method.